apps/passport-server/src/services/devconnectPretixSyncService.ts

```diff
@@ -135,14 +134,16 @@ export class DevconnectPretixSyncService {
   ): Promise<void> {
     return traced(NAME, "syncSingleOrganizer", async (span) => {
       try {
-        return await organizer.run();
+        await organizer.run();
       } catch (e) {
         logger(
           `[DEVCONNECT PRETIX] Error encounted when synchronizing organizer ${id}`,
           e
         );
         setError(e, span);
-        this.rollbarService?.reportError(e);
+        this.rollbarService?.reportError(
+          new Error("Devconnect Pretix Sync failed", { cause: e })
+        );
```

Comment:
maybe:

```suggestion
        this.rollbarService?.reportError(
          new Error("Devconnect Pretix Sync of organizer ${id} failed", { cause: e })
        );
```

Reply:
I've modified the `SyncFailureError` to include the organizer ID, so it will get logged with each item. I'm hopefully that we can log everything under a single heading ("Devconnect Pretix Sync failed") and then disaggregate it using queries on these fields, if we need to drill down into which organizers are failing.